thirdparty_builtin/benchmark-1.8.0/CMakeLists.txt

```diff
@@ -192,6 +192,9 @@ else()
     # See #631 for rationale.
     add_cxx_compiler_flag(-wd1786)
   endif()
+  if (CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
```

Comment:
Thanks @rchen20 . Can you follow the pattern we have with patches so this does not get lost when we upgrade Google Benchmark?

Here is an example:

https://github.com/LLNL/blt/blob/develop/thirdparty_builtin/patches/gbenchmark-2023-05-10-remove-minimum-cmake.patch

Reply:
@white238 If I add a diff patch, would I just remove the changes I've made directly in the CMakeLists.txt?

Reply:
leave in the changes but add a patch so i know they should be there.. i can also push a change if you like.

Reply:
Added the patch file. Let me know if you need anything else.